content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -232,3 +232,48 @@ $ kubectl scale --replicas=0 deploy/istio-citadel -n istio-system
 {{< /text >}}
 
 This should stop Istio from restarting Envoy and disconnecting TCP connections.
+
+## Multiple host gateway with SSL requirement
```

Comment:
The heading remains unclear. I suggest:
\#\# Configure a multiple host gateway with SSL